javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -699,3 +694,10 @@ async function getExternalPorts(
 
   return ports;
 }
+
+// enable --prometheus-external by default
+const prometheusExternal = (networkSpec: any): boolean => {
```

Comment:
```suggestion
// TODO: fix the `any` to an actual interface
const prometheusExternal = (networkSpec: any): boolean => {
```

Reply:
Yes, better to track this as `TODO`.
Thanks!